docs/new-audits.md

```diff
@@ -12,19 +12,24 @@ Lighthouse audits that surface in the report should be:
 - Measurable (especially for performance audits) or have clear pass/fail states. 
 - Not use 3rd party APIs for completing the audit check. 
 
-*Note: it's possible to submit a new audit proposal for something that you as a contributor feel very passionate about, but the Lighthouse team feels that it wouldn't add value to a significant portion of the Lighthouse users. In this case, we'd be open to having your audit be a part of the report when run with the full config on the Command Line.* 
 
 ## Process for creating a new audit
 1. Briefly scan the criteria we’ve laid out above. If you think the principles match with your proposed new audit, then proceed! 
 2. Next step is to create an issue on GitHub with answers to the following questions: 
-   - Description of audit and audit category (please include pass/fail states, and how it might be measured)
-   - Explanation of how it’s different from other audits
-   - What % of developers/pages will this impact (estimates OK, data points preferred) 
-   - How is the new audit making a better web for end users? (data points preferred)
-   - What is the resourcing situation (who will create the audits, maintain the audits, and write/maintain the documentation)
-   - Do you envision this audit in the Lighthouse report or the full config in the CLI? If in the report, which section? 
-   - How much support is needed from the Lighthouse team?
-   - Any other links or documentation that we should check out?
+```
+#### Provide a basic description of the audit
+#### How would the audit appear in the report? 
+  - What detailed info is shown in all cases, or just failure cases?
```

Comment:
this is a weird way of phrasing this, but I don't have a great alternative that's also short. What's the goal here?
- what are all the possible states (pass and fail) of the test results
- what information should be shown in each case to help the user improve
?
(but phrased better :)